docs/quickstart/index.rst

```diff
@@ -138,9 +140,17 @@ Set a password for lemur user inside Postgres:
 .. code-block:: bash
 
     $ sudo -u postgres psql postgres
+    \CREATE USER lemur WITH PASSWORD 'lemur';
+    \GRANT ALL PRIVILEGES ON DATABASE "lemur" to lemur;
     \password lemur
     Enter new password: lemur
     Enter it again: lemur
+    \connect lemur
+    create table encrypted_keys (ID int primary key not null);
```

Comment:
I also had to 

create table encrypted_passwords (ID int primary key not null);

since that was was missing as well.